crash-things.sh

```diff
@@ -21,35 +22,37 @@ CRASHER_CMD=(
 )
 
 echo "${CRASHER_CMD[*]}"
+
+QDRANT__LOGGER__ON_DISK__ENABLED=true \
+QDRANT__LOGGER__ON_DISK__LOG_FILE=$QDRANT_LOG \
```

Comment:
does it mean the logs are not pushed to std_out anymore with this option?

Reply:
No. The logs will be printed both to stdout *and* to the file. And we can tweak log levels, so that, e.g., stdout only logs `info` or `error`, but file logs `debug`.